src/index/ExternalSortFunctors.h

```diff
@@ -47,41 +47,22 @@ struct SortTriple {
       return cGraph < 0;
     }
   }
-
-  // Value that is strictly smaller than any input element.
-  static T min_value() { return {Id::min(), Id::min(), Id::min()}; }
-
-  // Value that is strictly larger than any input element.
-  static T max_value() { return {Id::max(), Id::max(), Id::max()}; }
 };
 
 using SortByPSO = SortTriple<1, 0, 2>;
 using SortByPSONoGraphColumn = SortTriple<1, 0, 2, false>;
 using SortBySPO = SortTriple<0, 1, 2>;
 using SortByOSP = SortTriple<2, 0, 1>;
 
-// TODO<joka921> Which of those are actually "IDs" and which are something else?
 struct SortText {
-  using T = std::tuple<TextBlockIndex, TextRecordIndex, WordOrEntityIndex,
-                       Score, bool>;
+  using Row = IdTableStatic<5>::row_type;
```

Comment:
This is a pessimization, because the `row_reference` is not `const Row&`.
So you should use `const auto&` as the argument.

Reply:
(You don't want to copy the rows in order to be able to compare them).